platform/security/security-migratable/src/main/java/org/codice/ddf/security/migratable/impl/SecurityMigratable.java

```diff
@@ -106,6 +110,42 @@ public void doExport(ExportMigrationContext context) {
 
   @Override
   public void doImport(ImportMigrationContext context) {
+    importPropertiesFiles(context);
+    importPdpPolicies(context);
+    LOGGER.debug(
+        "Importing security policy Directory at [{}]...", SecurityMigratable.SECURITY_POLICIES_DIR);
+    context.getEntry(SecurityMigratable.SECURITY_POLICIES_DIR).restore();
+  }
+
+  @Override
+  public void doVersionUpgradeImport(ImportMigrationContext context, String migratableVersion) {
+    if (Float.parseFloat(migratableVersion) > Float.parseFloat(getVersion())) {
```

Comment:
❓Should we catch `NumberFormatException` and assume that we're dealing with an incompatible version?

Reply:
❓This code is repeated three times. Should we consider moving it to a common class? Maybe create a new `Versions` class or something under the common migratable package and put all the code related to version extraction, transformation and validation in there?

Reply:
Yes, I can do that.